index.js

```diff
@@ -1,22 +1,30 @@
-// tooling
-const postcss = require('postcss');
-const parser = require('postcss-value-parser');
+const postcss = require("postcss");
```

Comment:
This should be single-quoted to be consistent.

```suggestion
const postcss = require('postcss');
```

Reply:
applied